packages/extension-api/src/extension-api.d.ts

```diff
@@ -2140,6 +2140,7 @@ declare module '@podman-desktop/api' {
     OpenStdin?: boolean;
     StdinOnce?: boolean;
     Detach?: boolean;
+    start?: boolean;
```

Comment:
if we add this to the API, doesn't this mean that we are now not 1-1 with the dockerode api?

Unsure if this is okay @benoitf

Reply:
it's ok dockerode will ignore the flag

Reply:
I think from user POV, it's better to have one option argument rather than two block of arguments, b/c they're all around creating a container